src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PhysicalPackageFile.cs

```diff
@@ -22,6 +22,8 @@ internal PhysicalPackageFile(Func<Stream> streamFactory)
             _streamFactory = streamFactory;
         }
 
+        public MemoryStream MemoryStream { get; set; } 
```

Comment:
this class is immutable. Consider changing this attribute to read-only along with providing relevant constructor.